src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts

```diff
@@ -241,13 +242,6 @@ export class ResultsFiltersPreferences extends Component {
       container.el.appendChild(valuesContainer.el);
 
       for (var i = 0; i < actives.length; i++) {
-        if (i != 0) {
```

Comment:
Was this loop not needed?

Reply:
Loop is needed, the separator (,) is not needed in between each element